### DISCUSSION TITLE
How to change the upper level config?

### DISCUSSION THREAD

**Discussion by Hannibal046** [facebookresearch/hydra#2651] [Q&A]
Hi, 
Thanks so much for this great projects! I am relatively new to Hydra, and I have a problem about **changing the upper level config**.

Suppose that my config folder looks like this:
<img width="225" alt="config yaml — hydra 2023-04-30 23-36-33" src="https://user-images.githubusercontent.com/38466901/235362109-c656c013-91c0-4745-9612-189bad5b5a33.png">

and `conf/config.yaml` looks like this:
<img width="241" alt="config yaml — hydra 2023-04-30 23-36-52" src="https://user-images.githubusercontent.com/38466901/235362136-5eabd92d-1dac-42ee-8fb8-0e77fb632269.png">

What i want is for `wmt` dataset, the `max_epochs` for 100 while for `iwslt` dataset, the `max_epochs` is 10. How could i implement this?

I know [Specializing configuration](https://hydra.cc/docs/patterns/specializing_config/), but this seems to handle a special case where there is only one should be specialized. But in my case, i may have much more datasets and i want to give each of them different trainer values, how could i achieve this?

**Comment by Kamichanw** [comment]
我也有一样的问题，你解决了吗

**Comment by zsh0613** [comment]
同样的问题 有解决方案吗
